### ISSUE TITLE
Test our autosuggest component with screen readers

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#5312]
Our autosuggest component, used for keywords and other metadata, has been built with WCAG compliance in mind. But it hasn't been tested with screen readers and not all screen readers can interpret some of the standard HTML components designed for such use cases (like a datalist).

We should test this component with an actual screen reader and if there are problems we should use [this guide](https://24ways.org/2019/making-a-better-custom-select-element/) to make adjustments. 

The key challenge for us is that we need to select multiple items, as well as support arbitrary values. But we should be able to adapt the recommended approach to use multiple `<input type="text">` fields for each entry.

**Comment by israelcefrin** [comment]
@NateWr I think we've tested it today. I will keep it open and below paste my comments:

> It was tested using voiceOver for Chrome, Firefox and Safari. Both had same issue: while I’m typing there’s no sound return about which key I’ve pressed. The user doesn’t know or can’t confirm whether it is being type. Also, when I’m typing in the URL address bar, safari warns me that I have a list of options to select using my down arrow. This warning doesn’t happen in this input field.
> Safari let me know what I’ve typed when I delete a text. Chrome or Firefox not even in this case.
